jaxb-ri/xjc/pom.xml

```diff
@@ -345,6 +345,22 @@
                             <goal>run</goal>
                         </goals>
                     </execution>
+                    <execution>
+                        <id>update-mr-jar</id>
+                        <phase>verify</phase>
+                        <configuration>
+                            <tasks>
+                                <jar destfile="${project.build.directory}/jaxb-xjc-${project.version}.jar" update="true">
+                                    <manifest>
+                                        <attribute name="Multi-Release" value="true"/>
```

Comment:
Shouldn't this be added by `maven-jar-plugin`?

Reply:
Good point, thanks @lukasj